### PR TITLE
Feature/footer-visibility

### PR DESCRIPTION
**Pull request by isaacyampo**
This commit changes the footer animation from a fade to a slide in and out. It also shortens the transition time

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -10,7 +10,9 @@
   import Button from '@smui/button'
   import Checkbox from '@smui/checkbox'
   import FormField from '@smui/form-field'
-    import { listen } from 'svelte/internal'
+  import { listen } from 'svelte/internal'
+  import { slide } from 'svelte/transition'
+	import { quintOut } from 'svelte/easing'
```
Comment (evansmj) [inline src/routes/+page.svelte:15]: indentation on this import

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -230,6 +241,20 @@
   function executeConnect() {
     pageViewModel.connect(address, rune, useTls)
   }
+
+  function handleMouseEnter() {
+    clearTimeout(timeoutId) // Clear any existing timeout
```
Comment (evansmj) [inline src/routes/+page.svelte:246]: can delete these 2 comments since these at least are self explanatory

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -230,6 +241,20 @@
...
+    isFooterVisible = true // make sure its visible on mouse enter
+  }
+
+  function handleMouseLeave() {
+    if (isDrawClicked) {
+      timeoutId = setTimeout(() => {
+        // Code to make the footer disappear after 5 seconds
```
Comment (evansmj) [inline src/routes/+page.svelte:253]: can delete these 2 comments also

**Review by evansmj — CHANGES_REQUESTED** [review]
looks good and the animation works nicely on chrome, some of the comments can be deleted though

File: src/routes/+page.svelte (hunk 1 above)
Comment (evansmj) [reply src/routes/+page.svelte:15]: looks like this indent is a tab and not spaces like the others?

**Review by isaacyampo — COMMENTED** [review]
Fixed indent and deleted some comments, as requested

**Review by evansmj — APPROVED** [review]

**Pull request merged**